core/DataTable/Renderer/Csv.php

```diff
@@ -246,6 +246,8 @@ protected function formatValue($value)
             $value = '"' . str_replace('"', '""', $value) . '"';
         }
 
+        //$value = str_replace(["\t"], ' ', $value);
```

Comment:
Guess you forgot to uncomment that one after testing. Also I guess it might be better to move that one before the previous if statement. The method is also used for TSV exports, where a tab might be tried to be escaped and then replaced. Might be smarter to first replace it, so it isn't escaped anymore.

```suggestion
        $value = str_replace(["\t"], ' ', $value);
```

Reply:
Thanks @sgiehl 
(Guess my code might work better if I didn't submit it commented out! :wink: )